agent-chat/public/app.css

```
@@ -150,6 +150,24 @@ button, [role="button"], .menu-item, .model-row, .row-control { cursor: default
+.model-picker-trigger-loading[data-visible="true"] {
+  width: 18px; opacity: 1; transform: scale(1); visibility: visible;
+  transition-delay: 80ms;
+  transition-timing-function: cubic-bezier(.16, 1, .3, 1);
+}
+.effort-picker-loading {
+  height: 24px; max-width: 150px; display: inline-flex; align-items: center; gap: 6px; padding: 0 7px;
+  color: var(--text-dim); background: color-mix(in srgb, var(--accent) 8%, transparent); border-radius: 6px;
+  font: 500 calc(var(--font-size-base) - 3px) var(--font-sans);
+  animation: loadingControlIn 130ms 80ms cubic-bezier(.16, 1, .3, 1) both;
+}
+@keyframes loadingControlIn { from { opacity: 0; transform: translateY(1px) scale(.96); } }
```

Stylelint (17.14.0)

[error] Expected keyframe name "loadingControlIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)